openstack/dns/v2/nameservers/results.go

```diff
@@ -0,0 +1,20 @@
+package nameservers
+
+import golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+
+type commonResult struct {
+	golangsdk.Result
+}
+
+type GetResult struct {
+	commonResult
+}
+
+// Extract is a function that accepts a result and extracts a nameserver.
+func (r commonResult) Extract() (*Nameserver, error) {
+	var s struct {
+		Nameserver *Nameserver `json:"nameserver"`
+	}
+	err := r.ExtractInto(&s)
```

Comment:
This should be done with `ExtractIntoStructPtr` with `nameserver` as a label instead of using local struct.